orcid-web/src/main/resources/freemarker/layout/public-layout.ftl

```diff
@@ -513,8 +513,9 @@
         '../common/maintenance_header.ftl'/> <#-- wtf --> <#nested>
     </div>
 </div>
+
 <!-- .container -->
-<#if RequestParameters['newFooter']??>
+<#if RequestParameters['NEW_FOOTER']?? || FEATURE['NEW_FOOTER']>
```

Comment:
I just created a macro for it, loot at the last commit https://github.com/ORCID/ORCID-Source/pull/3693/commits/4f25fcd46ce036efe90a73a2cecc3c8909a6c777